src/utils/config.cpp

```diff
@@ -120,7 +121,41 @@ void Config::ParseEmbrakes(char* line)
   }
 }
 
-void Config::ParseSensors(char* line)
+// if there is no creator configured to an interface, we use this one to prevent calling
+// a null pointer function
+template<class T>
+T* createDefault()
+{
+  printf("ERROOOR: no creator for %s found, creating NULL object\n", interfaceName<T>());
+  return NULL;
```

Comment:
```suggestion
  return nullptr;
```
Any reason for using `NULL`? Using `nullptr` (which is a keyword in C++) is preferred over `NULL` (which is just a macro for the literal integer `0`) as far as I understand. Some justifications are [here](https://www.learncpp.com/cpp-tutorial/6-7a-null-pointers/) and more can be found with a quick google